examples/intkey_go/Dockerfile-installed-bionic

```diff
@@ -30,7 +30,7 @@ RUN echo "deb [arch=amd64] http://repo.sawtooth.me/ubuntu/ci bionic universe" >>
  && apt-get update \
  && apt-get install -y -q \
     build-essential \
-    golang-1.9-go \
+    golang-1.10-go \
```

Comment:
Do we increment the go version, without actual need for it. Having lower version is helpful as long as upgrading to latest version arises. Backward compatibility is difficult as we move to latest versions. What do you think of it?

Reply:
Hello, I am upgrading to go 1.10 to deal with this mocgen issue. https://github.com/golang/mock/issues/273

Reply:
Thanks, I wasn't aware of it.